src/utils/createHashString.ts

```diff
@@ -0,0 +1,3 @@
+export default (length = 8) => {
+  return Math.random().toString(16).substr(2, length);
```

Comment:
Silly hash generator, we can change the specs as needed: 
- hex
- up to 14 length
- currently 8

Reply:
sure... but, more `leftpad`? it just feels wrong.